File: app/src/main/java/com/theone/simpleshare/ui/cocclient/CocClientViewModel.java
package com.theone.simpleshare.ui.cocclient;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class CocClientViewModel extends ViewModel {

    private MutableLiveData<String> mText;

    public CocClientViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue("This is notifications fragment");
    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: app/src/main/kotlin/com/theone/simpleshare/ui/cocclient/CocClientViewModel.kt
package com.theone.simpleshare.ui.cocclient

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class CocClientViewModel : ViewModel() {
    private val mText: MutableLiveData<String>
    val text: LiveData<String>
        get() = mText

    init {
        mText = MutableLiveData()
        mText.value = "This is notifications fragment"

    }
}